system_test/helpers/aest_nodes.erl

```diff
@@ -66,7 +66,11 @@
 %% When `backend` is `aest_docker`:
 
     % The source of the docker image
-    source  := {pull, binary() | string()}
+    source  := {pull, binary() | string()},
+    % Public/private peer key can be specified explicity for the node.
+    % Both are required and will be saved, overriding any present keys.
+    pubkey := binary(),
```

Comment:
Shall this be => ?

Reply:
Indeed: http://erlang.org/doc/reference_manual/typespec.html

Reply:
Yes indeed, my bad.